.github/workflows/aur-prerelease.yml

```diff
@@ -0,0 +1,24 @@
+name: AUR Prerelease
+
+on:
+  release:
+    types: [prereleased]
```

Comment:
I was looking into GitHub Actions as well recently.
This will trigger once a pre-release is published, right?
So this might happen already when I only wrote the description, but Drone didn't push the binaries yet, right?
Not that big of an issue as long as I think of it everytime though :man_shrugging: 
Also, I'm still waiting for Drone to be able to pick up drafts and convert them to real releases.. that would fix this as well.

Reply:
Yes, at least I understand it like that! I don't think any other of those `types` is suited for us (TBH github docs are pretty bad in this case, still not quite sure when each of them is triggered exactly). Ideally yea, drone would use the draft and publish after the binaries are uploaded.